test/test_jit_fuser_te.py

```diff
@@ -886,16 +959,26 @@ def test_lstm_traced(self):
             ge = self.checkTrace(LSTMCellF, inputs)
             graph = ge.graph_for(*inputs)
             fusion_groups = self.findFusionGroups(graph)
-            self.assertEqual(len(fusion_groups), 1)
-            FileCheck().check("Chunk").check("aten::sigmoid").check("aten::tanh").run(str(fusion_groups[0]))
+            # TODO: chunk
+            fusion_group_len = 2 if self.dynamic_shapes else 1
+            self.assertEqual(len(fusion_groups), fusion_group_len)
+            f = FileCheck()
+            if not self.dynamic_shapes:
+                f.check("Chunk")
+            f.check("aten::sigmoid").check("aten::tanh").run(str(fusion_groups[0 if not self.dynamic_shapes else 1]))
 
     def test_milstm(self):
+        if self.dynamic_shapes:
+            self.skipTest("don't run conv with dynamic shapes")
+
         for device in self.devices:
             inputs = get_milstm_inputs(device, training=True)
             module = self.checkScript(MiLSTMCell, inputs)
             forward_graph = module.graph_for(*inputs)
+            # TODO: chunk
+            fusion_group_len = 2 if self.dynamic_shapes else 1
```

Comment:
Why is the fusion group len for static shapes 1?

Reply:
there is a chunk that isn't being picked up in fusion